test/Spec.hs

```diff
@@ -15,13 +15,15 @@ import Test.Common (joinSpecs)
 import Test.Core.Issue (issueRoundtripProp)
 import Test.Core.Repo (repoRoundtripProp)
 import Test.Db (dbSpecs)
+import Test.Sync (syncSpecs)
 
 import qualified Data.Pool as Pool
 
 
 hspecTests :: AppEnv -> Spec
 hspecTests = sequential . joinSpecs "issue-wanted" 
     [ dbSpecs
+    , syncSpecs 
```

Comment:
When you don't need `AppEnv` in some of your tests, it's better to write `const syncSpecs` instead of `_` and specifying argument type.